src/lib/expansion/accordion.ts

```diff
@@ -0,0 +1,39 @@
+import {Directive, Input} from '@angular/core';
+import {
+  trigger,
+  state,
+  style,
+  transition,
+  animate,
+} from '@angular/animations';
+import {coerceBooleanProperty} from '../core/coercion/boolean-property';
+import 'rxjs/add/operator/takeUntil';
+
+export type CdkAccordionDisplayMode = 'default' | 'flat';
+
+/** Unique ID counter */
+let nextId = 0;
+
+/**
+ * Directive whose purpose is to manage the expanded state of CdkAccordionItem children.
+ */
+@Directive({
+  selector: '[cdk-accordion]',
```

Comment:
The idea for it being an attribute rather than an element is because it is simply a containing item for the children who control there own spacing and styling.  Since there is nothing in this directive that affects the document flow, it seemed to make sense that you could just mark another element as the containing element for the children.